src/Infrastructure/Mesh/src/Legacy/ESMCI_MeshPNC.C

```diff
@@ -122,7 +122,7 @@ void LoadNCDualMeshPar(Mesh &mesh, const std::string fname) {
       local_grid_start = my_start*grid_dims[0];
     }
     
-    int local_grid_dims[2] = {grid_dims[0], my_num > 0 ? my_num+1 : 0};
+    int local_grid_dims[2] = {grid_dims[0], (int) my_num > 0 ? (int) my_num+1 : 0};
```

Comment:
@theurich this change is not in develop (and likely the one below) so must not have been in the fix/cce branch that was merged that superseded this PR. I am confused about where we need these explicit casts? Or is @jedwards4b using some different compiler flags than you were when you had successful builds with CCE?

Reply:
@jedwards4b also tested successfully (https://github.com/esmf-org/esmf/pull/33#issuecomment-1128031723). The fix/cce branch never included these typecasts because I did not need them under my testing with CCE, and then Jim confirmed (see above link) that he also could compile without them. 
Notice however, that there were similar explicit typecast that were in fix/cce, two different files however. Not sure why those were needed, but the once in this PR were not at the time under our testing... and why now @jedwards4b has a system where they are needed again... really confusing.

Reply:
Could it be the cce version - this is using 14.0.2 which I believe is the latest available.

Reply:
Maybe. I think I am just going to merge both explicit the typecasts into develop, and have you try again. 
Once I have clean local testing after the change, I will push it, and tag it.. Then will let you know.

Reply:
thanks

Reply:
Part of v8.4.0b11 noiw.